lib/host/wasi/wasimodule.cpp

```diff
@@ -65,6 +65,9 @@ WasiModule::WasiModule() : ImportObject("wasi_snapshot_preview1") {
   addHostFunc("sock_recv", std::make_unique<WasiSockRecv>(Env));
   addHostFunc("sock_send", std::make_unique<WasiSockSend>(Env));
   addHostFunc("sock_shutdown", std::make_unique<WasiSockShutdown>(Env));
+  addHostFunc("sock_get_error", std::make_unique<WasiSockGetError>(Env));
```

Comment:
Since these functions are from linux socket, you should use the same name and function prototypes.